### PR TITLE
N122 feature fe auth headers added

### PR DESCRIPTION
**Pull request by dan933**

### CODE REVIEW

**Comment by dan933** [comment]
I apologise for the 100 console.logs thought it would be easier to work with if you could see the JWT but you probably needed 1 not 100 😝happy to remove some if you want.

File: nba-api-dotnet/appsettings.json [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
     "EveDesktopDB": "Data Source=(localdb)\\Project_DB;Initial Catalog=NBA;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False",
 
     "RikiLaptopDB":"Data Source=RIKIPC;Initial Catalog=NBA;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False",
     "WadeLaptopDB": "Data Source=LAPTOP-98R9H465;Initial Catalog=NBA;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False",
-    "AzureDatabase": "Server=tcp:nba-instance-one.database.windows.net,1433;Initial Catalog=NBA;Persist Security Info=False;User ID=nbaAdmin;Password=nbaPrediction_1;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;",
+    "AzureDatabase": "1Server=tcp:nba-instance-one.database.windows.net,1433;Initial Catalog=NBA;Persist Security Info=False;User ID=nbaAdmin;Password=nbaPrediction_1;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;",
```
Comment (EveJoyceStudent) [inline nba-api-dotnet/appsettings.json]: again

File: nba-front-end/src/components/Default.tsx [hunk 2]
```diff
@@ -8,6 +8,8 @@ const linkStyle = {
 };
 
 function DefaultPage() {
+
+  console.log(process.env);
```
Comment (EveJoyceStudent) [inline nba-front-end/src/components/Default.tsx]: again

**Review by EveJoyceStudent — CHANGES_REQUESTED** [review]
duplicates a bunch of stuff from the db change, so has the same issues. I assume most of that will just do nothing in the merge. The fe API changes look good though.

**Review by EveJoyceStudent — APPROVED** [review]
👍

**Pull request merged**